x-pack/metricbeat/module/statsd/server/data_test.go

```diff
@@ -839,7 +839,9 @@ func TestBuildMappings(t *testing.T) {
 	} {
 		t.Run(test.title, func(t *testing.T) {
 			var mappings []StatsdMapping
-			err := yaml.Unmarshal([]byte(test.input), &mappings)
+			if err := yaml.Unmarshal([]byte(test.input), &mappings); err != nil {
+				t.Error(err)
```

Comment:
Since this test is using assertions, for consistency purposes it might be better to use `assert` or `require`.

Reply:
Yes, I agree. Thanks!